internal/livelint/check_can_access_app.go

```diff
@@ -82,9 +83,26 @@ func (n *Livelint) canPortForward(pod apiv1.Pod, port int32) bool {
 
 	// send some traffic via the port forwarding
 	forwardedPorts, _ := forwarder.GetPorts()
-	_, err = net.Dial("tcp", fmt.Sprintf("localhost:%d", forwardedPorts[0].Local))
-	if err != nil {
-		return false
+	switch protocol {
+	case apiv1.ProtocolTCP,
+		apiv1.ProtocolUDP:
+		_, err = net.Dial(strings.ToLower(string(protocol)), fmt.Sprintf("localhost:%d", forwardedPorts[0].Local))
+		if err != nil {
+			return false
```

Comment:
If an error occurs and we have a verbose mode, it might be nice to log the actual dial error here. Also in the cases below for SCTP. That could give the user a hint as to why the forward did not work. It might be a mis-configuration on the server side and not the forwarding itself that fails.